inst/rmarkdown/templates/measles_outbreak/skeleton/skeleton.Rmd

```diff
@@ -1282,65 +1538,60 @@ inc_week_7_gender <- incidence(linelist_cleaned$date_of_onset,
                                interval = "Monday week", 
                                groups = linelist_cleaned$sex)
 
-# plot your epicurve
-# here we remove the boxes around each case as it makes gender colours hard to see! (show_cases = FALSE)
-plot(inc_week_7_gender, show_cases = FALSE, border = "black", n_breaks = nrow(inc_week_7_gender)) + 
+plot(inc_week_7_gender, 
+     show_cases = FALSE, # this removes black borders
+     border = "black", 
+     n_breaks = nrow(inc_week_7_gender)) + 
   scale_y_continuous(expand = c(0,0)) +  # set origin for axes
-  # add labels to axes and below chart
   epicurve_labels +
-  # change visuals of dates, remove legend title and move legend to bottom
   epicurve_theme
-```
-
-
-You could similarly stratify by vaccination status (or any other categorical variable!)
-
-```{r incidence_by_vax_status, message = FALSE}
```

Comment:
annick + isidro both asked for this, pls put back

Reply:
is this only for measles, or for measles/meningitis/cholera?

Reply:
only for ones they were in already